src/util.ts

```diff
@@ -170,6 +170,7 @@ export async function readConfigs(configPaths: string[]): Promise<UnifiedConfig>
       (prev, curr) => ({ ...prev, ...curr.commandLineOptions }),
       {},
     ),
+    sharing: configs.at(-1)?.sharing,
```

Comment:
Would it make sense to do something like
```ts
sharing: !configs.some(config => config.sharing === false),
```
So that if _any_ config has sharing disabled, the merged config won't have sharing either?

Reply:
This would be better!
I will fix it